experimental/s2ef_mapping.py

```diff
@@ -0,0 +1,22 @@
+import ase.io
+
+
+old_systems = glob.glob("path_to_old_systems/*.traj")
+new_system_path = "path_to_new_systems"
+
+system_mappings = {}
+for system in old_system:
+    randomid = os.path.basename(system)[:-5]
+    old_images = ase.io.read(system, ":")
+    old_energies = [image.get_potential_energy() for image in old_iamges]
+
+    new_images = ase.io.read(f"{new_system_path}/{randomid}.traj", ":")
+    new_energies = [image.get_potential_energy() for image in new_iamges]
+
+    dict_mapping = {}
+    for old_idx, energy in enumerate(old_energies):
```

Comment:
I was also thinking we can just `argsort` both new/old energies and then that will be a direct 1-1 mapping. Not sure if any edge cases there though.

Reply:
actually leaving it as is would serve as an extra validation things are the same